apps/generer-navn-service/src/main/java/no/nav/registre/testnorge/generernavnservice/service/VerifyNavnService.java

```diff
@@ -0,0 +1,25 @@
+package no.nav.registre.testnorge.generernavnservice.service;
+
+import no.nav.registre.testnorge.libs.dto.generernavnservice.v1.NavnDTO;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+import static java.util.Objects.isNull;
+
+@Service
+public class VerifyNavnService {
+    private static final List<String> ADJEKTIVER = FileReader.readLinesFromResources("static/adjektiv.csv");
+    private static final List<String> ADVERBER = FileReader.readLinesFromResources("static/adverb.csv");
+    private static final List<String> SUBSTANTIVER = FileReader.readLinesFromResources("static/substantiv.csv");
+
```

Comment:
Litt mange linger med AND og OR statements. Blir denne lettere å lese med?

```java
private boolean verify (String value, List<String> approvedValues){
       return isNull(value) || approvedValues.stream().anyMatch(validValue -> validValue.equalsIgnoreCase(value)))
}
public boolean verifyNavn(NavnDTO navnDTO) {
        return verify(navnDTO.getAdjektiv(), ADJEKTIVER) && verify(navnDTO.getAdverb(), ADVERBER) && verify(navnDTO.getSubstantiv(), SUBSTANTIVER)
}

```

Reply:
👍  Godt forslag!